ui/src/views/TestRunner.tsx

```diff
@@ -156,15 +156,17 @@ export const TestRunner = () => {
       const sepArray = testRunArrayCopy.find(({ sep }) => sep === test.sep);
       if (sepArray) {
         const testRun = sepArray.tests[testRunOrderMap[getTestRunId(test)]];
-        if (result) {
-          if (result.failureMode) {
-            sepArray.progress.failed++;
-          } else {
-            sepArray.progress.passed++;
+        if (testRun) {
```

Comment:
@Ifropc I'm not sure if this is a correct fix, but at least it won't break the UI. It looks like the tests are still running, so we could look for ways to not run them at all.

Reply:
Looks good! Getting rid of delay would be nice, but it's ok for now